src/libraries/System.Threading.Tasks.Extensions/tests/AsyncValueTaskMethodBuilderTests.cs

```diff
@@ -399,7 +399,7 @@ async ValueTask<int> ValueTaskReturningAsyncMethod(int result)
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
If the test is hanging, shouldn't that be an ActiveIssue?

Reply:
I've looked at it and it hangs in the threadpool pump loop in xharness as soon as we hit the "Incomplete" tests. I didn't have time to investigate more deeply but I agree, we should file an ActiveIssue for these cases.

Reply:
I created an ActiveIssue here https://github.com/dotnet/runtime/issues/38931